operator/src/main/java/oracle/kubernetes/operator/helpers/WebhookHelper.java

```diff
@@ -281,30 +264,57 @@ protected NextAction onFailureNoRetry(Packet packet,
                                             CallResponse<V1ValidatingWebhookConfiguration> callResponse) {
         LOGGER.info(MessageKeys.CREATE_VALIDATING_WEBHOOK_CONFIGURATION_FAILED,
             VALIDATING_WEBHOOK_NAME, callResponse.getE().getResponseBody());
-        return isNotAuthorizedOrForbidden(callResponse)
-            ? doNext(packet) : super.onFailureNoRetry(packet, callResponse);
+        return super.onFailureNoRetry(packet, callResponse);
       }
     }
 
     ResponseStep<V1ValidatingWebhookConfiguration> createReplaceResponseStep(Step next) {
       return new ReplaceResponseStep(next);
     }
 
-    private class ReplaceResponseStep extends ResponseStep<V1ValidatingWebhookConfiguration> {
+    private class ReplaceResponseStep extends WebhookConfigResponseStep {
       ReplaceResponseStep(Step next) {
         super(next);
       }
 
       @Override
       public NextAction onFailure(Packet packet, CallResponse<V1ValidatingWebhookConfiguration> callResponse) {
-        return super.onFailure(getConflictStep(), packet, callResponse);
+        if (UnrecoverableErrorBuilder.isAsyncCallNotFoundFailure(callResponse)) {
```

Comment:
The essence of this change is to NOT give the conflict step for non-recoverable errors. The code before this change gives a conflict step for all failures. We need to check 'Not Found" case before checking non-recoverable (in the super) because 404 is considered non-recoverable.